samples/sample-04/src/main/kotlin/com/remusrd/notesample/domain/annotation/NoArg.kt

```diff
@@ -0,0 +1,5 @@
+package com.remusrd.notesample.domain.annotation
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.SOURCE)
+annotation class NoArg
```

Comment:
Why do we need this annotation and the next `Open` one?
Aren't there something already explicit in Kotlin world?
Like I see those in the Gradle config:
```
apply plugin: "kotlin-allopen"
apply plugin: "kotlin-noarg"
```